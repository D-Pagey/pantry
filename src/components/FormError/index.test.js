import React from 'react';
import { FormError } from '.';

describe('FormError component', () => {
    it('should render', () => {
        const { container } = render(<FormError>Label</FormError>);
        expect(container.firstChild).toMatchSnapshot();
    });
});
