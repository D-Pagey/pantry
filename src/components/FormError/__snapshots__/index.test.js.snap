// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormError component should render 1`] = `
.c0 {
  color: red;
  font-size: 14px;
  margin: 0;
  padding: 0.5rem 0;
}

<p
  class="c0"
>
  Label
</p>
`;
